src/Vatsim/composer.json

```diff
@@ -0,0 +1,35 @@
+{
+    "name": "socialiteproviders/vatsim",
+    "description": "VATSIM OAuth2 Provider for Laravel Socialite",
+    "license": "MIT",
+    "keywords": [
+        "laravel",
+        "socialite",
+        "oauth",
+        "oauth2",
+        "provider",
+        "vatsim"
+    ],
+    "authors": [
+        {
+            "name": "Roy de Vos Burchart",
+            "email": "dev@bonroyage.com",
+            "role": "Developer"
+        }
+    ],
+    "support": {
+        "issues": "https://github.com/socialiteproviders/providers/issues",
+        "source": "https://github.com/socialiteproviders/providers",
+        "docs": "https://socialiteproviders.com/vatsim"
+    },
+    "require": {
+        "php": "^7.4 || ^8.0",
```

Comment:
```suggestion
        "php": "^8.0",
```

Reply:
I noticed that (as far as I saw) every provider supported PHP 7.4 and used manager 4.0, so it made sense not to deviate from that. I've updated the dependencies though.